.travis.yml

```diff
@@ -55,9 +55,9 @@ matrix:
       addons:
         postgresql: "9.4"
         sauce_connect: true
-  # Build artifacts only (no testing)
   allow_failures:
     - env: TOXENV=ui
+    - env: TOXENV=build
 
```

Comment:
`build` is a little ambiguous - can we rename that target (line 50 iiuc) to say `docker_build`?

Reply:
what about `build_artifacts` - technically we're building a deb package *and* a docker image

Reply:
great - i think that's more clear.